mysql/lib/dialect/mysql_database_dialect.ts

```diff
@@ -118,17 +118,14 @@ export class MySQLDatabaseDialect implements DatabaseDialect {
 
   async isClientValid(targetClient: ClientWrapper): Promise<boolean> {
     try {
-      return await ClientUtils.queryWithTimeout(
-        targetClient
-          .query({ sql: "SELECT 1" })
-          .then(() => {
-            return true;
-          })
-          .catch(() => {
-            return false;
-          }),
-        targetClient.properties
-      );
+      return await targetClient
```

Comment:
Is queryWithTimeout not necessary here?

Reply:
we set timeout parameters in query() already, but will revert the changes and keep setQueryTimeout because it has other things going on like deprecated properties checks etc